sections/slideshow.liquid

```diff
@@ -133,20 +133,18 @@
               assign height = block.settings.image.width | divided_by: block.settings.image.aspect_ratio | round
               if section.settings.image_behavior == 'ambient'
                 assign sizes = '120vw'
-                assign widths = '450, 660, 900, 1320, 1800, 2136, 2400, 3600, 7680'
+                assign widths = '450, 660, 900, 1320, 1800, 2136, 2400, 3600, 4608'
```

Comment:
similar thing here. Let me know if the `7680` was meant for a specific use case.